packages/salesforce-adapter/e2e_test/utils.ts

```diff
@@ -189,9 +189,10 @@ export const createElement = async <T extends InstanceElement | ObjectType>(
     changes: [{ action: 'add', data: { after: element } }],
   }
   const result = await adapter.deploy({ changeGroup, progressReporter: nullProgressReporter })
-  if (verify && result.errors.length > 0) {
-    if (result.errors.length === 1) throw result.errors[0]
-    throw new Error(`Failed adding element ${element.elemID.getFullName()} with errors: ${result.errors}`)
+  const errors = result.errors.filter(error => error.severity !== 'Warning')
```

Comment:
```suggestion
  const errors = result.errors.filter(error => error.severity === 'Error')
```
Otherwise we will face the same issue for `SaltoError` with severity `Info`

Reply:
Ack